bin/release

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+# shellcheck source=functions
+if [[ "$(basename "$0")" == release ]] ; then
+  source "$(dirname "$0")/../functions"
+else
+  source "$(dirname "$0")/../sanctuary-scripts/functions"
+fi
+
+declare -r increment="$1"
+owner="$(get repo-owner)"
+name="$(get repo-name)"
+prefix="$(get version-tag-prefix)"
+
+node_modules/.bin/xyz \
+  --increment "${increment}" \
+  --repo "git@github.com:${owner}/${name}.git" \
+  --script scripts/prepublish \
```

Comment:
Good idea. I'll make this change. We can keep `scripts/prepublish` as the default value, and have `check-required-files` assert that the (explicitly or implicitly) specified file is present and executable.